plugins/doc_fragments/purestorage.py

```diff
@@ -11,17 +11,17 @@
 class ModuleDocFragment(object):
 
     # Standard Pure Storage documentation fragment
-    DOCUMENTATION = r'''
-options:
-  - See separate platform section for more details
+    DOCUMENTATION = r"""
+options: {}
+# See separate platform section for more details
 requirements:
   - See separate platform section for more details
 notes:
-  - Ansible modules are available for the following Pure Storage products: FlashArray, FlashBlade
-'''
+  - 'Ansible modules are available for the following Pure Storage products: FlashArray, FlashBlade.'
+"""
```

Comment:
There are 3 doc fragments with this "abstract-base-fragment" construct, which IMHO makes little sense. Should we keep those?

Reply:
Why do you think they make little sense?

Reply:
Testing the theory in #9433